array_test.go

```diff
@@ -1283,3 +1283,116 @@ func TestArray_NotFind(t *testing.T) {
 		array.chain.assertFailed(t)
 	})
 }
+func TestArray_IsOrdered(t *testing.T) {
+	type args struct {
```

Comment:
Let's also add a test for the case when httpexpect assertion fails inside comparator function.

For example, array contains a number, and comparator invokes value.String().